samples/net/azure_iot_hub/src/main.c

```diff
@@ -4,37 +4,42 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#include <zephyr/kernel.h>
+
 #include <stdio.h>
 #include <stdlib.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/reboot.h>
+#include <zephyr/net/net_if.h>
```

Comment:
Not nessecary I think, net_if is indirectly included in conn_mgr and there should be no need to call net_if calls.